docs/nodes/nodes-library/nodes/Slack/README.md

```diff
@@ -0,0 +1,67 @@
+---
+permalink: /nodes/n8n-nodes-base.slack
+---
+
+# Slack
+
+[Slack](https://slack.com) is a business communication platform offering many IRC-style features, including persistent chat rooms (channels), private groups, and direct messaging.
+
+The instructions for authenticating with Slack are [here](../../../credentials/Slack/).
```

Comment:
This needs to be formatted as in the other nodes. An example: https://github.com/n8n-io/n8n-docs/tree/master/docs/nodes/nodes-library/nodes/Twilio